go/ct/utils/adapter_test.go

```diff
@@ -163,7 +163,7 @@ func TestAdapter_ParameterConversion(t *testing.T) {
 			},
 			func(p tosca.Parameters) (any, any) {
 				ctxt := p.Context
-				return tosca.Word{}, ctxt.GetCommittedStorage(tosca.Address{}, tosca.Key{})
+				return tosca.Word{}, ctxt.GetCommittedStorage(tosca.Address{}, tosca.Key{}) //nolint:staticcheck
```

Comment:
It would be nice to comment what issues should be ignored here, that apparently can not be fixed.

Reply:
So far, all ignores are deprecated calls. Fixing these deprecations is quite some work. 
Let me annotate a fix up issue.

Reply:
- added issue #616, so that we can enable lint checks in PR asap, fixing these deprecations is delicate work.
- commented ignored lint lines